### DISCUSSION TITLE
There should be a link to this repo from header or footer of the Docs site.

### DISCUSSION THREAD

**Discussion by rikh42** [livewire/flux#676] [Feature Requests]
Since this is where discussion and support is handled, there should be an easy to find link straight here.

**Comment by GuidoVeltmaat** [comment]
the support  on [https://fluxui.dev/account](https://fluxui.dev/account) is now e-mail, ive used it and got a reply to come here.

**Reply by rikh42** [reply]
me too.

**Comment by joshhanley** [comment]
@rikh42 and @GuidoVeltmaat yeah good call, I've made a note of it 🙂
